src/main/java/org/isf/operation/gui/OperationRowAdm.java

```diff
@@ -409,15 +421,15 @@ public void addToForm() {
 			textFieldUnit.setText(opeRow.getTransUnit() + ""); //$NON-NLS-1$
 		}
 
-		/* ***** resultat **** */
-		int index = -1;
+		/****** resultat *****/
+		int index = 0;
 		for (int i = 0; i < operationResults.size(); i++) {
-			if (opeRow.getOpResult() != null && (operationResults.get(i) + "").equals(opeRow.getOpResult())) { //$NON-NLS-1$
+			if (opeRow.getOpResult() != null && (opeManager.getResultDescriptionKey(operationResults.get(i) + "")).equals(opeRow.getOpResult())) { //$NON-NLS-1$ 
 				index = i;
 			}
 		}
-		comboResult.setSelectedIndex(index + 1);
-		/* *********** */
+		comboResult.setSelectedIndex(index);
+		/*************/
```

Comment:
Please undo this type of change as JavaDoc analyzers view the current line 425 as the start of a dangling JavaDoc comment. Thanks.